index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">
+        freeCodeCamp Survey Form
+        </h1>
+        
+        <p id="description">
+          Thank you for taking the time to help us improve the platform
+        </p>
+        
+        <form id="survey-form">
+        
+          <label id="name-label">
+          Name:<input type="text" id="name" required placeholder="enter your name">
+          </label>
+          <br><br>
+        <label id="email-label">
+          email:<input type="email" id="email" required placeholder="enter your email">
+          </label>
+          <br><br>
+          <label id="number-label">
+          Age:<input type="number" id="number" min="0" max="100" placeholder="enter your age">
+          </label>
+ <select id="dropdown">
+   <option>Option 1</option>
+   <option>Option 2</option>
+   <option>Option 3</option>
+   </select> <br><br>
+<input type="radio" id="name" value="value"
+name="radio1"><label>Definitely</label><br/>
+<input type="radio" id="name" value="value" name="radio1"><label>Maybe</label><br/>
```

Comment:
Ensure that you do not duplicate `id` on the elements as they are meant to be unique
```jsx
id="name"
```

Reply:
For the next reviewer this comment was submitted after submitting for review